static/src/styles/PostCard.styl

```diff
@@ -1,20 +1,75 @@
 .PostCard {
-  background: $Color__bright1;
-  padding: grid(2);
+  background: $Color__bright2;
+  background-size: cover;
   border-radius: $Layout__normalBorderRadius;
   min-height: grid(50);
   color: $Color__lightText;
+  position: relative;
+  overflow: hidden;
+  z-index: 0;
+  cursor: pointer;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
 }
 
-.PostCard__label {
+.PostCard__photoBackground {
```

Comment:
I avoided the blend mode because I figure in general, legibility will be better if we just do an opacity thing. Take a look at the result and see how you feel.